xAPI.md

```diff
@@ -509,11 +509,47 @@ __Verb__: Defines the action being done by the Actor within the Activity within
 
 ## 8.0 Getting More Out of xAPI
 
+xAPI focuses on the structure of communication between a client and an LRS.  It spares many details of the content to
+increase flexibility.  The intention in only supplying a basic amount of syntax rules is that Communities of Practice (CoP) can define content (verbs, activity types, contextual relationships, extensions, etc.) for their use cases.  It is very 
+important that such communities exist and share best practices.
+
 ### 8.1 Extending xAPI
 
-### 8.2 Profiles
+xAPI can be extended in a few ways.  The most notable and extensive is Statement Extensions, which allow great flexibility 
+within Statements.  It is recommended that profiles or Communities or Practice agree on how to use extensions for their 
+particular use cases. Implementation details are covered in a <a href="#miscext">later section</a>.  
+
+The About Resource is another way xAPI enables extensions.  The LRS may find it useful to communicate features or 
+behaviors beyond this specification to activity provider.  It is hoped that if more information is available about an 
+LRS, that the About Resource captures it.
+
+Finally, the set of Resources implemented is not expected to be constrained by this document. While not explicitly 
+defined, this document suggests that no new Endpoints defined in future iterations of this specification will 
```

Comment:
Good idea, the point of this last sentence to talk about how xAPI is extended, not exceptions to extending.